cs/build.sh

```diff
@@ -4,29 +4,36 @@ PROTOCOL_DIR="$( dirname $SCRIPT_DIR )"
 
 pushd $PROTOCOL_DIR
 
+# generate codecs
```

Comment:
This build.sh looks like a workaround to `generator.py`. 
Can we move it to `generator.py` so that all other clients may benefit?

Reply:
looking into it - might kill it instead as it's rather CS specific